### PR TITLE
Documentation improvements

### PR DESCRIPTION
**Pull request by nothendev**
This PR includes documentation for the WebSocket API for realtime monitoring and interaction with Pterodactyl servers and a README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+## Articles
+The articles are an extension of the OpenAPI spec,
+and are additional documentation for, for example, the WebSocket API.
+Currently there is one article:
+- [WebSocket API](./docs/Server-Websockets.md)
+
+# Libraries
+This list includes libraries that help developers interact with the Pterodactyl API. \
+More entries will be added soon.
+
```
Comment (devnote-dev) [inline README.md:20]: This is not what was meant by including API libraries/wrappers...

File: reference/Client-API.yaml [hunk 2]
```diff
@@ -1725,12 +1725,7 @@ paths:
                   signal: stop
               properties:
                 signal:
-                  type: string
```
Comment (devnote-dev) [inline reference/Client-API.yaml:1728]: Why was this changed?

**Review by devnote-dev — DISMISSED** [review]

**Comment by devnote-dev** [comment]
As a side note: all the websocket information will eventually be ported from [devnote-dev/ptero-notes](https://github.com/devnote-dev/ptero-notes/blob/main/wings/websocket.md), so it would be best to leave that out for now.

File: README.md (hunk 1 above)
Comment (nothendev) [reply README.md:20]: Then please tell me what you actually ment.

File: reference/Client-API.yaml (hunk 2 above)
Comment (nothendev) [reply reference/Client-API.yaml:1728]: Didn't know Stoplight hasn't reverted that

**Comment by nothendev** [comment]
> As a side note: all the websocket information will eventually be ported from [devnote-dev/ptero-notes](https://github.com/devnote-dev/ptero-notes/blob/main/wings/websocket.md), so it would be best to leave that out for now.

Do you want me to pull that? Or just delete that?

File: README.md (hunk 1 above)
Comment (devnote-dev) [reply README.md:20]: Having a dedicated page in the docs for API libraries/wrappers, but there still needs to be more discussion on what repositories get added to it.

**Comment by devnote-dev** [comment]
> Do you want me to pull that? Or just delete that?

Delete it for now, it will be added later.

File: README.md (hunk 1 above)
Comment (nothendev) [reply README.md:20]: For now, this would be a section in readme. There aren't enough good wrappers. For now it'd just be dartactyl

Comment (TekExplorer) [reply README.md:20]: we could potentially consider adding a section somewhere that mentions all known wrappers and how much of the API they cover (ex application, client, includes websocket wrapper, etc) and sort them on the page based on that

**Review by TekExplorer — COMMENTED** [review]
Honestly didn't realize this was still here.

Good enough to start
